src/frontend/js/components/ReferralForm/TopicField.tsx

```diff
@@ -158,16 +171,45 @@ export const TopicField: React.FC<TopicFieldProps> = ({
           <TopicSuggestion {...{ topic, isHighlighted }} />
         )}
         shouldRenderSuggestions={() => true}
+        renderSuggestionsContainer={({ containerProps, children, query }) => (
+          <div {...containerProps}>
+            {children}
+
+            {!children &&
+              query.length > 0 &&
+              state.context.value.length === 0 &&
+              isInputFocused && (
+                <div>
```

Comment:
```suggestion
                <div className="p-4">
```

I think some padding would look better here.